packages/suite-desktop/package.json

```diff
@@ -175,6 +175,7 @@
     "dependencies": {
         "@sentry/electron": "3.0.0",
         "@trezor/connect": "9.0.0-beta.1",
+        "@trezor/suite-analytics": "1.0.0",
```

Comment:
nit: for local packages it's better to use *

Reply:
2cb63179f535f7394215d597c07fd06f269c7b9a